spec/models/event_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  subject {Event.new(title: "Anything",teaser: "Lorem ipsum",bar: "Panorama Bar",day: "Friday",date: "30.09.2022",websource: ["berlinevent","visitevent"])}
```

Comment:
@moizansari22 One main spec is missing.You need to create event object with valid values and check the if event object is created in DB or not.

Reply:
resolved